Sources/SKOptions/SourceKitLSPOptions.swift

```diff
@@ -76,6 +76,9 @@ public struct SourceKitLSPOptions: Sendable, Codable, Equatable {
     ///   background indexing.
     public var skipPlugins: Bool?
 
+    /// Whether to use SwiftPM's experimental build server in place of SourceKit-LSP's builtin package support..
```

Comment:
```suggestion
    /// Whether to use SwiftPM's experimental build server in place of SourceKit-LSP's builtin package support.
```

(nit-pick - feel free to include with some later change)

I was also considering `useExperimentalBuildServer` here since it's already under SwiftPM, though "SwiftPMBuildServer" is technically more accurate. Don't have a strong opinion either way.